scripts/ufm_fabric_health/README.md

```diff
@@ -0,0 +1,83 @@
+UFM Fabric Health
+--------------------------------------------------------
+
+
+This script is used to run Fabric Health Report via the UFM APIs.
```

Comment:
I would suggest to have a brief description for the fabric health reports 
and a link for more details under :
https://docs.nvidia.com/networking/display/UFMEnterpriseUMv69/Fabric+Health+Tab